components/script/dom/window.rs

```diff
@@ -1553,7 +1556,8 @@ impl Window {
         // TODO Step 1
         // TODO(mrobinson, #18709): Add smooth scrolling support to WebRender so that we can
         // properly process ScrollBehavior here.
-        self.layout_chan
+        self.layout_chan()
+            .unwrap()
```

Comment:
We'll want to replace these unwraps in this PR with warn! statements and return from the code as soon as possible.